tests/__fixtures__/Rules/newline_before_scenario/fixture.js

```diff
@@ -0,0 +1,559 @@
+  Rule: a rule
+
+    @tag1
+
+    @tag2
+    Scenario Outline: a scenario outline`,
+            [generateProblem({ line: 7, column: 5 }, 2)],
+        ],
+        [
+            "with Rule: Scenario comments",
+            `Feature: a feature file
+  Rule: a rule
+    #comment line
+    Scenario: a scenario`,
+            [generateProblem({ line: 4, column: 5 }, 0)],
+        ],
+        [
+            "with Rule: description - Scenario tags and comments",
+            `Feature: a feature file
+  feture description here
+  Rule: a rule
+    @tag1
+
+    #comment line
+
+    @tag2
+
+    Scenario: a scenario`,
+            [generateProblem({ line: 10, column: 5 }, 3)],
+        ],
+        [
+            "with multiple Rules: description - Scenario tags and comments",
+            `Feature: a feature file
+  feture description here
+  Rule: a rule
+    @tag1
+
+    #comment line
+    @tag2
+
+    Scenario: a scenario
+
+  Rule: a rule
+    #comment line
+    @tag1 @tag2
+    Scenario: a scenario`,
+            [
+                generateProblem({ line: 9, column: 5 }, 2),
+                generateProblem({ line: 14, column: 5 }, 0),
+            ],
+        ],
+        [
+            "without Rule: Scenario",
+            `Feature: a feature file
+  Scenario: a scenario`,
+            [generateProblem({ line: 2, column: 3 }, 0)],
+        ],
+        [
+            "without Rule: Background - Scenario",
+            `Feature: a feature file
+  Background: a background
+
+
+  Scenario: a scenario`,
+            [generateProblem({ line: 5, column: 3 }, 2)],
+        ],
+        [
+            "without Rule: Scenario Outline",
+            `Feature: a feature file
+  Scenario Outline: a scenario outline`,
+            [generateProblem({ line: 2, column: 3 }, 0)],
+        ],
+        [
+            "without Rule: Background - Scenario Outline",
+            `Feature: a feature file
+  Background: a background
+  Scenario Outline: a scenario`,
+            [generateProblem({ line: 3, column: 3 }, 0)],
+        ],
+        [
+            "without Rule: multiple scenarios",
+            `Feature: a feature file
+  Background: a background
+
+
+  Scenario: a scenario
+
+
+
+  Scenario Outline: a scenario outline`,
+            [
+                generateProblem({ line: 5, column: 3 }, 2),
+                generateProblem({ line: 9, column: 3 }, 3),
+            ],
+        ],
+        [
+            "without Rule: description - Scenario",
+            `Feature: a feature file
+  Some feature description
+  is here
+  Scenario: a scenario`,
+            [generateProblem({ line: 4, column: 3 }, 0)],
+        ],
+        [
+            "without Rule: Scenario tags",
+            `Feature: a feature file
+
+
+  @tag1 @tag2
+
+  Scenario: a scenario`,
+            [generateProblem({ line: 6, column: 3 }, 3)],
+        ],
+        [
+            "without Rule: Scenario multiline tags",
+            `Feature: a feature file
+
+  @tag1
+
+  @tag2
```

Comment:
An idea for another rule, maybe called `no_lines_between_tags`
That would fail for this case.

Reply:
yeah, added to https://github.com/gherlint/gherlint/issues/17